turbo/src/components/report/InfoPanelClaim.svelte

```diff
@@ -1,32 +1,36 @@
 <script lang="ts">
-	import _ from 'lodash';
 	import { _ as __ } from 'svelte-i18n';
 
+	import { HHMMSSToSeconds } from '$lib/utils';
+	import Player from './Player.svelte';
+
 	export let showVideo: boolean;
 	export let claim: any;
 	export let csv: any;
 	export let showClaims: boolean;
 
-	let newIframeSrc: string;
+	let videoSrc: string;
 	let claimCommentId: string = claim.commentId;
```

Comment:
I don't really know how this InfoPanel is instanced, but I would have `$claimCommentId = claim.commentId` in case the claim changes.

Reply:
Agreed